### ISSUE TITLE
got an exception:  ValueError: too many values to unpack

### ISSUE THREAD

**Issue by apexchu** [MitchellChu/torndsession#2]
# I got an exception when the application 24H after running. detail as below:

Traceback (most recent call last):
File "/usr/local/lib/python2.7/site-packages/tornado-4.0-py2.7-linux-i686.egg/tornado/web.py", line 1332, in _execute
result = method(self.path_args, *self.path_kwargs)
File "/usr/www/haiyo/src/haiyo/ajaxhandlers.py", line 192, in post
self.finish()
File "/usr/local/lib/python2.7/site-packages/tornado-4.0-py2.7-linux-i686.egg/tornado/web.py", line 899, in finish
self.on_finish()
File "/usr/local/lib/python2.7/site-packages/torndsession-1.1.2-py2.7.egg/torndsession/sessionhandler.py", line 26, in on_finish
self.session.flush() # try to save session
File "/usr/local/lib/python2.7/site-packages/torndsession-1.1.2-py2.7.egg/torndsession/session.py", line 185, in flush
self.driver.save(self._id, self.session, self._expires)
File "/usr/local/lib/python2.7/site-packages/torndsession-1.1.2-py2.7.egg/torndsession/memorysession.py", line 48, in save
self.remove_expires()
File "/usr/local/lib/python2.7/site-packages/torndsession-1.1.2-py2.7.egg/torndsession/memorysession.py", line 58, in remove_expires
for key, val in self._data_handler:
ValueError: too many values to unpack

**Comment by MitchellChu** [comment]
Oops.
memorysession.py has a bug in remove_expires(self) method. you need to fix this bug in current version.
fixed code:

``` python
    def remove_expires(self):
        for key, val in self._data_handler.items():
            now = datetime.datetime.utcnow()
            expires = val.get("__expires__", now)
            if now > expires:
                del self._data_handler[key]
```

**Comment by MitchellChu** [comment]
this problem has been fixed in new version(1.1.3), thank you for your feedback.
